_posts/2024-03-06-firo-slogan-showdown.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "The Firo Slogan Showdown"
+summary: "Unleash Your Creativity!"
+tags: [Community]
+author: "Batuhan Yıldırım"
+img: "/blog/assets/sloganshowdown/banner.png"
+---
+Attention Firofam!
+
+Do you have a knack for creativity and a way with words? We're thrilled to announce the Firo Slogan Showdown! We're looking for a catchy, impactful slogan that reflects the core message of our project and uplifts the values in our society. The new slogan can be proudly displayed on our project's T-shirts or used as a tagline for the Firo community.
+
+Join us in this exciting challenge to craft a slogan that resonates with our mission of privacy, security, and innovation. Your idea could be the next iconic statement for Firo!
+
+As part of the event, an award will be given to the participant who comes up with the most creative and meaningful slogan. If you're curious about what the award might be, we welcome you to join our event filled with surprises!
+
+## Event Details
+
+* **Start Date**: 6th March, 2024 
+* **Deadline for Submission**: 20th March 2024 
+
+### **How to Participate** 
+
+You can submit your entry with the hashtag **#FiroSloganShowdown** in the official [Firo Telegram Channel](https://t.me/firoorg), or reply to the Twitter/X post. 

```

Comment:
We could also mention the matrix room since it's bridged with telegram

Reply:
Sure